pkgs/development/compilers/llvm/11/compiler-rt/default.nix

```diff
@@ -25,19 +25,20 @@ stdenv.mkDerivation {
     "-DCOMPILER_RT_DEFAULT_TARGET_ONLY=ON"
     "-DCMAKE_C_COMPILER_TARGET=${stdenv.hostPlatform.config}"
     "-DCMAKE_ASM_COMPILER_TARGET=${stdenv.hostPlatform.config}"
-  ] ++ lib.optionals (useLLVM || isDarwin || bareMetal || isMusl) [
+  ] ++ lib.optionals (useLLVM || bareMetal || isMusl ||
+                      (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64)) [
```

Comment:
I think we can just do this! Everything else I think we just beating around the bush.
```suggestion
  ] ++ lib.optionals haveLibc [
```

Reply:
Sorry I edited this a few times, I'll think about it and do another.

Reply:
Won’t this disable the sanitizers on some platforms where they work, notably x86_64-darwin?

Reply:
I think after all my second-guessing and editing -- no :).

Reply:
Is that comment intentional?